### ISSUE TITLE
Act doesn't work on path where in path is "space" character.

### ISSUE THREAD

**Issue by invoodoo** [richardszalay/fastlane-plugin-act#6]
unzip couldn't find this path, it should be encoded for it.

```
/Users/InVooDoo/Library/Mobile Documents/com~apple~CloudDocs/iWork/Fancy Interactive/GlobeConvert/GlobeConvert/Free.ipa
```

**Comment by richardszalay** [comment]
@invoodoo Thanks for the catch. I'm not working on a Mac at the moment, but I'd happily accept a PR.

**Comment by richardszalay** [comment]
Note for implementation (whether it be you, or me when I get time), the change is basically just calling `"/path/with spaces/".shellescape`

**Comment by richardszalay** [comment]
This fix has been released as v1.1.1
